### PR TITLE
Features/style card

### PR DESCRIPTION
**Pull request by Jacksonmcguire**
## What does this PR do?

- Provides initial styling for the card container, card, and filter form. Minor refactors to the filter form. Creates and styles the sidebar component

## How should it be tested in the UI?

- Messing around with the sidebar component and filter form component on a browser to see if they behave as expected.

### CODE REVIEW

File: src/components/FilterForm/FilterForm.tsx [hunk 1]
```diff
@@ -29,7 +36,6 @@ export class FilterForm extends React.Component<FilterProps> {
 
   handleChange = (e: any):void => {
```
Comment (itsnameissteven) [inline src/components/FilterForm/FilterForm.tsx:37]: I found a way that we can avoid using any here, Ill include it in my branch

File: src/components/SideBar/SideBar.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.side-bar {
+  float: left;
+  height: 100%;
+  position:fixed;
+  background-color: whitesmoke;
+  transform: translate3d(-88%, 0px, 0px);
```
Comment (itsnameissteven) [inline src/components/SideBar/SideBar.css:6]: x,y,z ??? This is cool, haven't seen this before

**Review by itsnameissteven — APPROVED** [review]
This looks good to me, I didn't have a lot to say about most the code because it's well done. I fetched this branch and tested locally and everything works well.

File: src/components/FilterForm/FilterForm.tsx (hunk 1 above)
Comment (Jacksonmcguire) [reply src/components/FilterForm/FilterForm.tsx:37]: Awesome, forgot about that one

File: src/components/SideBar/SideBar.css (hunk 2 above)
Comment (Jacksonmcguire) [reply src/components/SideBar/SideBar.css:6]: ya definitely cool and don't know a ton about it, but you can do some cool 3d animations with it [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/translate3d())

**Pull request merged**